src/plugin/regions/region.js

```diff
@@ -611,4 +612,15 @@ export class Region {
             });
         }
     }
+
+    updateHandlesResize(resize) {
+        const cursorStyle = resize ? 'col-resize' : 'none';
+        const childNodes = this.element.childNodes;
+
+        childNodes.length > 0 && childNodes.forEach(child => {
+            if (child.nodeName === 'HANDLE') {
+                this.style(child, {...child.style, 'cursor': cursorStyle});
```

Comment:
no need for `...child.style` here.  `this.style` only sets new style key/values, see src/util/style.js

Reply:
Yes, you're right, thanks, I'm fixing this